src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Telemetry/UpgradeInformationEvent.cs

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Linq;
+using NuGet.Common;
+using NuGet.ProjectManagement;
+
+namespace NuGet.PackageManagement.Telemetry
+{
+    public class UpgradeInformationTelemetryEvent : TelemetryEvent
```

Comment:
https://github.com/NuGet/NuGet.Client/blob/dev/docs/coding-guidelines.md#code-comments

Reply:
This should be internal anyway. I was going off the pattern of other classes in same folder, which are all comment-less-public. However, it seems that all of them can be converted to internal?